From 62cbfb4ceb41884cd3563b700af7d46fb99572aa Mon Sep 17 00:00:00 2001
From: Ira Weiny <ira.weiny@intel.com>
Date: Wed, 11 Mar 2015 01:28:42 -0400
Subject: [PATCH] infiniband-diags/rdma-ndd: Fix issues with install

On Debian in particular it was reported that the binary was attempting to run
from /sbin when it was installed to /usr/sbin.  This was not seen on RHEL as
RHEL defines /sbin to be a softlink to /usr/sbin.

In addition fix the /var path by using the localstatedir macro.

Reported-by: Dan Ben Yosef <danby@mellanox.com>
Signed-off-by: Ira Weiny <ira.weiny@intel.com>
---
 etc/rdma-ndd.init.in | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/etc/rdma-ndd.init.in b/etc/rdma-ndd.init.in
index 41747f20ec..24ce1bdb1c 100644
--- a/etc/rdma-ndd.init.in
+++ b/etc/rdma-ndd.init.in
@@ -41,7 +41,9 @@
 # notice, one of the license notices in the documentation
 # and/or other materials provided with the distribution.
 
-pidfile=/var/run/rdma-ndd.pid
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+pidfile=@localstatedir@/run/rdma-ndd.pid
 
 # Source function library.
 if [[ -s /etc/init.d/functions ]]; then
-- 
2.4.3

